src/commands/validate.ts

```diff
@@ -69,6 +70,7 @@ export default class Validate extends SleekCommand {
       addonName = flags.addonName;
     }
     let skipHooksValidation = flags.skipHooks;
+    let skipReleaseService = flags.skipResourceService;
```

Comment:
is above is changed, this needs too :) also my intellij suggests to use const instead of let as it's never reassigned

Reply:
tks, this comment flagged other error, i missed support skipping this when the input is an input yaml file